test/index.test.js

```diff
@@ -54,6 +54,23 @@ test('env values override', function(t) {
   t.end();
 });
 
+test('env values strings are trimmed', function(t) {
+  process.env.SNYK_whitespace = ' string '; // jshint ignore:line
```

Comment:
suggest to also add a whitespace in the middle of the string to show it's not deleted